pkg/ingester/ingester.go

```diff
@@ -1344,6 +1344,18 @@ func (i *Ingester) pushSamplesToAppender(userID string, timeseries []mimirpb.Pre
 			return true
 
 		// Map TSDB native histogram validation errors to soft errors.
+		case errors.Is(err, storage.ErrOOONativeHistogramsDisabled):
+			stats.invalidNativeHistogramCount++
```

Comment:
I'm not entirely sure this counts as an invalid native histogram. Technically, the histogram could have correct we just don't support accepting them.

Reply:
Agree this is not invalid, rather `stats.sampleOutOfOrderCount++`

Reply:
good shout, done in https://github.com/grafana/mimir/pull/9567/commits/911760a898e2a029d472577254bd2e764e3f0ca7